package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "playcanvas-spine",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
Let's bump version in a separate PR. I normally use the `npm version` command to bump package version instead of doing it by hand.

Reply:
Oh yes, for the tag and release commit. 

Does it do the lock file too or do you need to do `npm install` ?

One moment.

Reply:
Completed in f171916